docs/azure/nRFCloudLocationServices.rst

```diff
@@ -0,0 +1,39 @@
+.. _azure-nrf-cloud-location-services:
+
+nRF Cloud Location Services
+###########################
+
+Cell Location Service
+*********************
+
+You can enable the :ref:`cell geolocation lookup <app-cellgeolocation>` for single cells and neighboring cell reports for the application using `nRF Cloud's Cell Location Service API <https://api.nrfcloud.com/v1#tag/Cell-Position>`_ in your deployment.
+
+To use nRF Cloud's Cell Location Service API, set the ``enableNrfCloudCellLocation`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
```

Comment:
```suggestion
To use API, set the ``enableNrfCloudCellLocation`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
```
I believe you don't need to repeat the full name of the API as you just mentioned that in the previous chapter.

Reply:
```suggestion
To use the API, set the ``enableNrfCloudCellLocation`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
```